deploy/aws-hypervisor/scripts/force-stop.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+SCRIPT_DIR=$(dirname "$0")
+source "${SCRIPT_DIR}/common.sh"
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Configuration for wait retries (shorter for force stop)
+FORCE_STOP_WAIT_RETRIES=${FORCE_STOP_WAIT_RETRIES:-2}
+
+# Function to wait for instance to stop with retry logic
+wait_for_instance_stopped() {
+    local instance_id="$1"
+    local attempt=1
+    
+    echo "Waiting for instance to stop (max ${FORCE_STOP_WAIT_RETRIES} attempts)..."
+    
+    while [[ $attempt -le $FORCE_STOP_WAIT_RETRIES ]]; do
+        echo "Attempt ${attempt}/${FORCE_STOP_WAIT_RETRIES}..."
+        
+        set +e  # Allow this command to fail
+        aws --region "${REGION}" ec2 wait instance-stopped --instance-ids "${instance_id}"
```

Comment:
I feel like this can cause some confusion, we should probably add the `--no-cli-pager` flag to this call, when you get dropped into the pager it can seem like you're supposed to do something.